crm_compassion/model/event_compassion.py

```diff
@@ -58,9 +58,12 @@ class event_compassion(models.Model):
     country_id = fields.Many2one('res.country', 'Country')
     user_id = fields.Many2one(
         'res.users', 'Ambassador', track_visibility='onchange')
+    available_children_ids = fields.One2many('compassion.hold',

```

Comment:
This should be renamed to `hold_ids` and you can add another field for `available_children_ids` that point to `compassion.child` records, like this:
```
available_children_ids = fields.One2many('compassion.child', related='hold_ids.child_id')
```